config/src/lib.rs

```diff
@@ -146,6 +146,11 @@ pub struct BlockSynchronizerParameters {
     /// discover who has the payload available for the dictated certificates.
     #[serde(with = "duration_format")]
     pub payload_availability_timeout: Duration,
+    /// When a certificate is submitted from the handler for further
+    /// processing, we wait for it to become available for us to use. This
+    /// is the timeout while we wait for it.
```

Comment:
See the comment on benchmark/README: this is another place to explain what is expected to happen in this wait.